### PR TITLE
add access + refresh token to jwt

### PR DESCRIPTION
**Pull request by johnnicholson**

### CODE REVIEW

File: webServer/src/main/java/com/jnicho/quickpick/Authentication/CognitoAuthenticationManager.java [hunk 1]
```diff
@@ -71,7 +71,10 @@ public Authentication authenticate(Authentication auth) throws AuthenticationExc
         }
         // Unsure if this is how you are supposed to confirm the login (check later)
         if (resp != null && isBlank(resp.getChallengeName())) {
-            return new AWSAuthToken(username, password, user.getAuthorities());
+            resp.getAuthenticationResult().getAccessToken();
```
Comment (mdavis60) [inline webServer/src/main/java/com/jnicho/quickpick/Authentication/CognitoAuthenticationManager.java]: why is this called?

File: webServer/src/main/java/com/jnicho/quickpick/models/User.java [hunk 2]
```diff
@@ -83,7 +83,7 @@ public boolean isAccountNonExpired() {
     @JsonIgnore
     @Override
     public boolean isAccountNonLocked() {
-        return !AWSAccount;
+        return true;
```
Comment (mdavis60) [inline webServer/src/main/java/com/jnicho/quickpick/models/User.java]: this smells funky...

Comment (johnnicholson) [reply webServer/src/main/java/com/jnicho/quickpick/models/User.java]: it's a spring boot required property that I tried to use cleverly, but shouldn't have originally

File: webServer/src/main/java/com/jnicho/quickpick/Authentication/CognitoAuthenticationManager.java (hunk 1 above)
Comment (johnnicholson) [reply webServer/src/main/java/com/jnicho/quickpick/Authentication/CognitoAuthenticationManager.java]: cause I'm a derp

**Pull request merged**